jsnlog/Infrastructure/LogMessageHelpers.cs

```diff
@@ -1,21 +1,21 @@
 ﻿using System;
 using System.Collections.Generic;
-using Newtonsoft.Json;
+using System.Text.Json;
 
 namespace JSNLog.Infrastructure
 {
     internal class LogMessageHelpers
     {
         public static T DeserializeJson<T>(string json)
         {
-            T result = JsonConvert.DeserializeObject<T>(json);
+            T result = JsonSerializer.Deserialize<T>(json);
             return result;
         }
 
         public static bool IsPotentialJson(string msg)
         {
             string trimmedMsg = msg.Trim();
-            return (trimmedMsg.StartsWith("{") && trimmedMsg.EndsWith("}")); 
+            return (trimmedMsg.StartsWith("{") && trimmedMsg.EndsWith("}"));
```

Comment:
Maybe you could revert these formatting changes (adding/removing whitespaces) from this file as well as the .nuspec files. Without it the PR is more concise and clear.

Reply:
@mperdeck to help this along should I bother or are you happy to review it with the whitespace changes? Effectively without the whitespace there is only 3 Addison's & 3 removals.